lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ChillerElectricEIR.rb

```diff
@@ -0,0 +1,81 @@
+class ASHRAE901PRM < Standard
+  # @!group ChillerElectricEIR
+
+  # Finds the search criteria
+  #
+  # @return [hash] has for search criteria to be used for find object
+  def chiller_electric_eir_find_search_criteria(chiller_electric_eir)
```

Comment:
The methods in this file are missing YARD comments for the parameters.

Reply:
Added